app/docs/0.12.x/admin-api.md

```diff
@@ -1787,6 +2025,111 @@ Attributes | Description
 HTTP 204 No Content
 ```
 
+---
+
+### Set target as healthy
```

Comment:
camel case for titles? (im not sure if we follow this pattern or not)

Reply:
The whole doc is a mess regarding title casing. Since "fixing" all of it would be outside of the scope of the PR, I just decided to follow the casing of "Target" section that this is under.